lb.tf

```
@@ -85,3 +85,30 @@ resource "aws_route53_record" "wildcard" {
 data "dns_a_record_set" "nlb" {
   host = module.nlb.lb_dns_name
 }
+
+
+locals {
+  # List of Node_Groups names from the module configuration which have "nlbs_attachment = true" or undefined (true by default)
```

```suggestion
  # List of Node_Groups names from the module configuration which have "nlbs_attachment = true" or undefined (true by default)
```

It's here you need to change if you put false by default :)